testing/ef-tests/src/cases/blockchain_test.rs

```diff
@@ -4,6 +4,7 @@ use crate::{
     models::{BlockchainTest, ForkSpec},
```

Comment:
Remaining changes in this files are mainly the invalid block feature (i.e., returning partial witnesses), and very initial feature of running tests returning witnesses.